Submodels/MTSOL_call.py

```
@@ -1,20 +1,613 @@
+        -------
+        None
+        """
+
+        if exit_flag == ExitFlag.NON_CONVERGENCE.value:
+            self.HandleNonConvergence()
+        elif exit_flag == ExitFlag.CRASH.value:
+            self.CrashRecovery(case_type,
+                               iter_count,
+                               )
+        elif exit_flag == ExitFlag.SUCCESS.value or exit == ExitFlag.NOT_PERFORMED.value:
+            return None
```

Ruff (0.8.2)

Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)